style.css

```diff
@@ -31,14 +31,61 @@
     display: none;
   }
 
-  .head-top a {
+  .mobile-nav {
+    position: inherit;
+    display: block;
+  }
+
+  .x-btn {
+    position: absolute;
+    right: 10%;
+    color: white;
+    font-size: 2.5rem;
+    z-index: 99;
+  }
+
+  .display-class {
+    display: none;
+  }
+
+  .bg-style {
+    background-color: rgba(96, 112, 255, 0.7);
+    backdrop-filter: blur(8px);
+    width: 100vw;
+    height: 100vh;
+    position: absolute;
+  }
+
+  .head-top-a {
     color: #6070ff;
     font-weight: 650;
     font-size: 1.2rem;
     text-decoration: none;
     align-self: center;
   }
 
+  .unhide-nav {
+    display: block;
+    position: absolute;
+    right: 10%;
+    color: white;
+    font-size: 2.5rem;
+    z-index: 99;
+  }
+
+  .main-nav ul {
+    list-style-type: none;
+    margin-right: 35%;
+  }
+
+  .main-nav ul li a {
+    text-decoration: none;
+    color: white;
+    font-size: 2.8rem;
+    font-weight: 600;
+    line-height: 44px;
+  }
```

Comment:
There are some slight differences between the menu items and that of the figma design as indicated below :point_down: . Kindly check the position, margin, and size and ensure that they match the figma design. 

### Your design

![Screenshot from 2023-01-03 15-54-11](https://user-images.githubusercontent.com/65233860/210394779-91ee22f9-23d5-4a6f-9c06-b6097cc13f3e.png)


### Figma design 

![Screenshot from 2023-01-03 15-54-18](https://user-images.githubusercontent.com/65233860/210394795-ae128337-94ed-4f91-97b6-a3ef35dce42b.png)